x/warp/contract.go

```diff
@@ -240,46 +231,33 @@ func UnpackGetVerifiedWarpMessageOutput(output []byte) (GetVerifiedWarpMessageOu
 	return outputStruct, err
 }
 
-func unpackWarpMessage(accessibleState contract.AccessibleState, input []byte) ([]byte, bool) {
+type messageHandler interface {
```

Comment:
could we add interface assertions for the types implementing this?

Reply:
Yup